src/System.Management.Automation/security/wldpNativeMethods.cs

```diff
@@ -1,3 +1,8 @@
+//
```

Comment:
Can you make sure that the copyright header is applied?

Reply:
Done